oauth_dropins/reddit.py

```diff
@@ -114,21 +115,21 @@ def get(self):
                          user_agent='oauth-dropin reddit api')
 
     refresh_token = reddit.auth.authorize(code)
-    user = reddit.user.me()
-    # a short list of attributes to grab, for more info on attributes see:
-    # https://github.com/praw-dev/praw/blob/master/praw/models/reddit/redditor.py#L41
-    # calling json_dumps on the user object opens some kind of stream
-    attribute_list = ['comment_karma',
-                      'created_utc',
-                      'id',
-                      'name',
-                      'link_karma',
-                      'icon_img']
-    user_json = {a:getattr(user,a) for a in attribute_list}
-    user_id = str(user)
+    praw_user = reddit.user.me()
+    user_json = praw_to_user(praw_user)
+    user_id = user_json.get('name')
 
     auth = RedditAuth(id=user_id,
                       refresh_token=refresh_token,
                       user_json=json_dumps(user_json))
     auth.put()
     self.finish(auth, state=st)
+
+def praw_to_user(user):
+  return {
```

Comment:
this removes `comment_karma` and `link_karma` and adds `subreddit`. i assume that's intentional?